assets/onboarding/data/actions.js

```diff
@@ -106,9 +106,11 @@ export const startSubmit = ( step, stepData ) => ( {
  * Success submit action creator.
  *
  * @return {{type: string}} Success submit action.
+ * @param {string} step Completed step.
```

Comment:
I think it would make more sense to add the params before the return. Do you agree?